.github/workflows/branch-rules.yaml

```
@@ -0,0 +1,24 @@
+
+on:
+    pull_request:
+        types: [opened, edited, synchronize, reopened]
+
+jobs:
+    branch-name-check:
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Check branch name
+              run: |
```

actionlint (1.7.11)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)